tests/Collection/CollectionFunctionalTest.php

```diff
@@ -61,27 +61,14 @@ public function testConstructorOptionTypeChecks(array $options): void
         new Collection($this->manager, $this->getDatabaseName(), $this->getCollectionName(), $options);
     }
 
-    public function provideInvalidConstructorOptions()
+    public function provideInvalidConstructorOptions(): void
```

Comment:
Why does this have a `void` return type? It looks like you're discarding the return value from `createOptionDataProvider()`.

Reply:
Good catch - not sure how that slipped through the cracks. I've updated this and the other occurrence and double-checked that I didn't make the same mistake more often.